### PR TITLE
Bootstrap the Bookmark plugin.

### PR DESCRIPTION
**Pull request by pszczesniak**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal (bookmark): Bootstrap the Bookmark plugin. Closes #000.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-bookmark/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+CKEditor&nbsp;5 bookmark feature
+================================
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-bookmark.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-bookmark)
+[![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
+[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
+
+## TODO
+This package contains the bookmark feature for CKEditor&nbsp;5. It allows to manage the bookmarks across the content of editor.
```
Comment (pszczesniak) [inline packages/ckeditor5-bookmark/README.md:9]: @godai78 Could you take a look at this description. please?

File: packages/ckeditor5-bookmark/ckeditor5-metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"plugins": [
+		{
+			"name": "Bookmark",
+			"className": "Bookmark",
+			"description": "Allows to insert a bookmark into the content. // TODO",
```
Comment (pszczesniak) [inline packages/ckeditor5-bookmark/ckeditor5-metadata.json:6]: [@godai78](https://github.com/godai78) Could you take a look at this description. please?

Comment (godai78) [reply packages/ckeditor5-bookmark/ckeditor5-metadata.json:6]: Looking at it.

File: packages/ckeditor5-bookmark/README.md (hunk 1 above)
Comment (pszczesniak) [reply packages/ckeditor5-bookmark/README.md]: Thank you @godai78

File: packages/ckeditor5-bookmark/ckeditor5-metadata.json (hunk 2 above)
Comment (pszczesniak) [reply packages/ckeditor5-bookmark/ckeditor5-metadata.json]: Thank you @godai78

**Pull request merged**